polyfill/lib/ecmascript.mjs

```diff
@@ -1763,8 +1765,15 @@ export const ES = ObjectAssign({}, ES2019, {
         days += ((seconds / 60 + minutes) / 60 + hours) / 24;
 
         const oneYear = new TemporalDuration(1);
+        const sign = Math.sign(days);
```

Comment:
the whole `oneYearDays`, `oneMonthDays` and `oneWeekDays` logic seems a bit complicated, but I'll take your word for it :smile:

Reply:
Do you think I should explain in a comment what this is for, or something?

Reply:
That would be ideal! :smile: